packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -17,7 +17,22 @@ import {
   UseAutocompleteProps,
   AutocompleteValue,
 } from "@mui/material";
-import { memo, useCallback, useMemo, useRef } from "react";
+import React, {
+  createContext,
+  FC,
+  forwardRef,
+  memo,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+} from "react";
+import { VariableSizeList, ListChildComponentProps } from "react-window";
+import _AutoSizer, { Props } from "react-virtualized-auto-sizer";
+
+// This is required to get around a react-types issue for "AutoSizer is not a valid JSX element."
+// @see https://github.com/bvaughn/react-virtualized/issues/1739#issuecomment-1291444246
+const AutoSizer = _AutoSizer as unknown as FC<Props>;
```

Comment:
Can we rename this import from the generic `Props` to `AutoSizerProps`?
```suggestion
import _AutoSizer, { Props as AutoSizerProps } from "react-virtualized-auto-sizer";

// This is required to get around a react-types issue for "AutoSizer is not a valid JSX element."
// @see https://github.com/bvaughn/react-virtualized/issues/1739#issuecomment-1291444246
const AutoSizer = _AutoSizer as unknown as FC<AutoSizerProps>;
```

This should help with any confusion from someone skimming the file especially during a PR (like me).

Reply:
Yep, good suggestion.